Gemfile

```diff
@@ -31,8 +31,8 @@ group :test do
   gem 'webmock', require: false
 end
 
-local_ast = File.expand_path('../rubocop-ast', __dir__)
-gem 'rubocop-ast', path: local_ast if Dir.exist? local_ast
+# local_ast = File.expand_path('../rubocop-ast', __dir__)
+# gem 'rubocop-ast', path: local_ast if Dir.exist? local_ast
```

Comment:
Please revert this to its original state.

Reply:
Whoops, that wasn't meant to be committed.